client/js/lounge.js

```diff
@@ -312,6 +312,15 @@ $(function() {
 		if (!target) {
 			return;
 		}
+
+		const targetTitle = self.data("title");
+		if (targetTitle.indexOf("#") === -1 && !self.hasClass("lobby")) { // no # or .lobby; assume it's a user query
```

Comment:
This entire thing can be done on the server inside `open`, and it can check if open channel type is query.